### PR TITLE
feat: update docs for edgex 3.0 update

### PR DESCRIPTION
**Pull request by vli11**
Closes: #137

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-vending/blob/main/.github/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented out code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?
update automated checking to automated vending; updated toml file to yaml file and their new links.

## Issue this PR will close

close: #137 

## Test Instructions if applicable
read through the docs_src folder, all automated checkout should be replaced, and no more toml file

### CODE REVIEW

File: docs_src/automated-vending-services/device_services.md [hunk 1]
```diff
@@ -265,12 +265,12 @@ The following picture illustrates the flow to calculate the inventory deltas bet
     <img src="../images/delta.png">
 </p>
 
-The Automated Checkout architecture uses three MQTT topics:
+The Automated Vending architecture uses three MQTT topics:
 
 | Topic                       | Description                                                                                             |
 | ----------------------------- | ------------------------------------------------------------------------------------------------ |
 | Inference/CommandTopic                  | All events pushed from EdgeX's command API are fed into this topic. The [`as-vending`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-vending) and [`as-controller-board-status`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-controller-board-status) are two services that make requests to this API, typically for making door close/open and heartbeat events. |
```
Comment (jim-wang-intel) [inline docs_src/automated-vending-services/device_services.md:272]: we probably also want to fix the url for that too: 
```suggestion
| Inference/CommandTopic                  | All events pushed from EdgeX's command API are fed into this topic. The [`as-vending`](https://github.com/intel-retail/automated-vending/tree/main/as-vending) and [`as-controller-board-status`](https://github.com/intel-retail/automated-vending/tree/main/as-controller-board-status) are two services that make requests to this API, typically for making door close/open and heartbeat events. |
```

File: docs_src/automated-vending-services/device_services.md [hunk 2]
```diff
@@ -265,12 +265,12 @@ The following picture illustrates the flow to calculate the inventory deltas bet
...
-| Inference/ResponseTopic  | The Automated Checkout cv inference service will respond to published messages on the `Inference/CommandTopic` topic on the `Inference/ResponseTopic` topic. |
+| Inference/ResponseTopic  | The Automated Vending cv inference service will respond to published messages on the `Inference/CommandTopic` topic on the `Inference/ResponseTopic` topic. |
 | Inference/DataTopic                | The cv inference service publishes delta SKUs on this topic, then the MQTT device service converts them into EdgeX event readings, and finally the [`as-vending`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-vending) service processes the event readings and pushes them to downstream services. |
```
Comment (jim-wang-intel) [inline docs_src/automated-vending-services/device_services.md:274]:
```suggestion
| Inference/DataTopic                | The cv inference service publishes delta SKUs on this topic, then the MQTT device service converts them into EdgeX event readings, and finally the [`as-vending`](https://github.com/intel-retail/automated-vending/tree/main/as-vending) service processes the event readings and pushes them to downstream services. |
```

File: docs_src/automated-vending-services/device_services.md [hunk 3]
```diff
@@ -279,7 +279,7 @@ The Automated Checkout architecture uses three MQTT topics:
 
 #### `GET`: `http://localhost:59982/api/v2/device/name/Inference-device/inferenceHeartbeat`
```
Comment (jim-wang-intel) [inline docs_src/automated-vending-services/device_services.md:280]:
```suggestion
#### `GET`: `http://localhost:59982/api/v3/device/name/Inference-device/inferenceHeartbeat`
```

File: docs_src/automated-vending-services/device_services.md [hunk 4]
```diff
@@ -279,7 +279,7 @@ The Automated Checkout architecture uses three MQTT topics:
...
 
-The `GET` call to the EdgeX MQTT device service's `inferenceHearbeat` command will act as a health-check for the Automated Checkout cv inference service. It must return `200 OK` upon swiping an RFID card in order for the vending workflow to begin. If it does not, the [`as-vending`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-vending) service will enter maintenance mode.
+The `GET` call to the EdgeX MQTT device service's `inferenceHearbeat` command will act as a health-check for the Automated Vending cv inference service. It must return `200 OK` upon swiping an RFID card in order for the vending workflow to begin. If it does not, the [`as-vending`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-vending) service will enter maintenance mode.
```
Comment (jim-wang-intel) [inline docs_src/automated-vending-services/device_services.md:282]: fix url to intel-retail

File: docs_src/configuration.md [hunk 5]
```diff
@@ -1,29 +1,13 @@
 # Configuration
 
-This page lists all of the relevant configuration parameters for each service in the Automated Checkout reference implementation.
+This page lists all of the relevant configuration parameters for each service in the Automated Vending reference implementation.
 
 !!!info
     Note that this document likely does not cover EdgeX-specific configuration parameters. Application and device service SDK documentation can be found in the [EdgeX Foundry GitHub repositories](https://github.com/edgexfoundry) or in the [official EdgeX documentation here](https://docs.edgexfoundry.org/2.2/).
```
Comment (jim-wang-intel) [inline docs_src/configuration.md:6]:
```suggestion
    Note that this document likely does not cover EdgeX-specific configuration parameters. Application and device service SDK documentation can be found in the [EdgeX Foundry GitHub repositories](https://github.com/edgexfoundry) or in the [official EdgeX documentation here](https://docs.edgexfoundry.org/3.0/).
```

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
there are some url still pointing to the old url (intel-iot-devkit) and /v2 to /v3
please do a global search under docs/src folder to fix them all.

File: docs_src/configuration.md [hunk 6]
```diff
@@ -1,29 +1,13 @@
...
-```yaml
-Driver_VID: "256" ** Good **
-Driver_PID: "26"  ** Good **
-
-DRIVER_VID: "256" ** BAD **
-driver_pid: "26"  ** BAD **
-```
-
-These overrides are placed in the target service's environment section of the compose file. Here is an example:
+The simplest way to change one of the configuration values described below is via the use of environment variable overrides in the docker compose file. The value of each configuration item in a service's configuration can be overridden with an environment variable specific to that item. The name of the environment variable is the path to the item in the configuration tree with underscores separating the nodes. The character case of each node in the environment variable name must match that found in the service's configuration. These overrides are placed in the target service's environment section of the compose file. Here is an example:
```
Comment (lenny-intel) [inline docs_src/configuration.md:10]: You removed the `values described below` need so reference/example here.

configuration.yaml
```yaml
Driver:
  VID: "65535" # 0xFFFF
  PID: "53"    # 0x0035
```

compose file
```yaml
DRIVER_VID: "256" 
DRIVER_PID: "26"  
```

File: docs_src/configuration.md [hunk 7]
```diff
@@ -58,19 +42,15 @@ The following items can be configured via the `DriverConfig` section of the serv
 
 ## EdgeX MQTT device service
 
-This reference implementation uses the [MQTT Device Service](https://github.com/edgexfoundry/device-mqtt-go) from EdgeX with custom device profiles. These device profiles YAML files are located [here](https://github.com/intel-iot-devkit/automated-checkout/blob/master/res/device-mqtt/profiles/inference-mqtt-device-profile.yml) and are volume mounted into the device service's running Docker container.
+This reference implementation uses the [MQTT Device Service](https://github.com/edgexfoundry/device-mqtt-go) from EdgeX with custom device profiles. These device profiles YAML files are located [here](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/res/device-mqtt/profiles/inference-mqtt-device-profile.yml) and are volume mounted into the device service's running Docker container.
```
Comment (lenny-intel) [inline docs_src/configuration.md:45]: This one exists on main, so link to main branch

File: docs_src/configuration.md [hunk 8]
```diff
@@ -58,19 +42,15 @@ The following items can be configured via the `DriverConfig` section of the serv
...
 
-The following items can be configured via `MQTTBrokerInfo` section of the service's [configuration.toml](https://github.com/intel-iot-devkit/automated-checkout/blob/master/res/device-mqtt/configuration.toml) file. All values are strings.
+The following items can be configured via `device-mqtt.environment` section of the service's [docker-compose.edgex.yml](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/docker-compose.edgex.yml) file.
```
Comment (lenny-intel) [inline docs_src/configuration.md:47]: compose file exist on main, so like to main branch

File: docs_src/modifying_source_code.md [hunk 9]
```diff
@@ -22,11 +22,11 @@ Next, build the specific service's image:
 make ds-card-reader
 ```
 
-After Docker builds the image (by executing the steps in [`ds-card-reader/Dockerfile`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ds-card-reader/Dockerfile)), proceed to the next section.
+After Docker builds the image (by executing the steps in [`ds-card-reader/Dockerfile`](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ds-card-reader/Dockerfile)), proceed to the next section.
```
Comment (lenny-intel) [inline docs_src/modifying_source_code.md:25]: file exists on main, link to main branch

File: docs_src/phases/phase1.md [hunk 10]
```diff
@@ -117,7 +117,7 @@ curl -X PUT -H "Content-Type: application/json" -d '{"card-number":"0003293374"}
     There should not be any response message when running this EdgeX command successfully.
 
 !!! info
-    By default, the card number `0003293374` corresponds to a card in the [`ms-authentication/cards.json`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-authentication/cards.json) file that has the "stocker" role associated to it.
+    By default, the card number `0003293374` corresponds to a card in the [`ms-authentication/cards.json`](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-authentication/cards.json) file that has the "stocker" role associated to it.
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:120]: file exists on main, link to main branch

File: docs_src/phases/phase1.md [hunk 11]
```diff
@@ -207,7 +207,7 @@ curl -X GET http://localhost:48095/inventory
   <summary><i>(Click to Expand)</i> Inventory API Response Example</summary>
 
 <p>
-The (altered) contents of <a href="https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-inventory/inventory.json"><code>ms-inventory/inventory.json</code></a> are contained in the <code>content</code> key below.
+The (altered) contents of <a href="https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-inventory/inventory.json"><code>ms-inventory/inventory.json</code></a> are contained in the <code>content</code> key below.
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:210]: file exists on main, link to main branch

File: docs_src/phases/phase1.md [hunk 12]
```diff
@@ -235,7 +235,7 @@ curl -X GET http://localhost:48095/auditlog
   <summary><i>(Click to Expand)</i> Audit Log API Response Example</summary>
 
 <p>
-The (altered) contents of <a href="https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-inventory/auditlog.json"><code>ms-inventory/auditlog.json</code></a> are contained in the <code>content</code> key below.
+The (altered) contents of <a href="https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-inventory/auditlog.json"><code>ms-inventory/auditlog.json</code></a> are contained in the <code>content</code> key below.
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:238]: file exists on main, link to main branch

File: docs_src/phases/phase1.md [hunk 13]
```diff
@@ -261,11 +261,11 @@ Since this a stocking event and not a customer transaction, there is no ledger e
 </p>
 
 <p>
-By default, the ledger service has six registered accounts (accounts 1-6) for consumers. The valid accounts match the account ID's that have authorized access to the cooler through the <code>ms-authentication</code> service (see the <a href="https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-authentication/accounts.json"><code>ms-authentication/accounts.json</code></a> file).
+By default, the ledger service has six registered accounts (accounts 1-6) for consumers. The valid accounts match the account ID's that have authorized access to the cooler through the <code>ms-authentication</code> service (see the <a href="https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-authentication/accounts.json"><code>ms-authentication/accounts.json</code></a> file).
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:264]: file exists on main, link to main branch

File: docs_src/phases/phase1.md [hunk 14]
```diff
@@ -261,11 +261,11 @@ Since this a stocking event and not a customer transaction, there is no ledger e
...
 </p>
 
 <p>
-The (unaltered) contents of <a href="https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-ledger/ledger.json"><code>ms-ledger/ledger.json</code></a> are contained in the <code>content</code> key below.
+The (unaltered) contents of <a href="https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-ledger/ledger.json"><code>ms-ledger/ledger.json</code></a> are contained in the <code>content</code> key below.
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:268]: file exists on main, link to main branch

File: docs_src/phases/phase1.md [hunk 15]
```diff
@@ -370,7 +370,7 @@ The following command makes a REST API call to the `ds-controller-board` service
 curl -X PUT -H "Content-Type: application/json" -d '{"setDoorClosed":"1"}' http://localhost:48097/api/v2/device/name/controller-board/setDoorClosed
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:370]:
```suggestion
curl -X PUT -H "Content-Type: application/json" -d '{"setDoorClosed":"1"}' http://localhost:48097/api/v3/device/name/controller-board/setDoorClosed
```

File: docs_src/phases/phase1.md [hunk 16]
```diff
@@ -384,7 +384,7 @@ A few items have been removed from the inventory in the below API response. Care
 </p>
 
 <p>
-The (altered) contents of <a href="https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-inventory/inventory.json"><code>ms-inventory/inventory.json</code></a> are contained in the <code>content</code> key below.
+The (altered) contents of <a href="https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-inventory/inventory.json"><code>ms-inventory/inventory.json</code></a> are contained in the <code>content</code> key below.
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:387]: file exists on main, link to main branch

File: docs_src/phases/phase1.md [hunk 17]
```diff
@@ -419,7 +419,7 @@ The audit log has a new transaction that corresponds to this customer interactio
 </p>
 
 <p>
-The (altered) contents of <a href="https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-inventory/auditlog.json"><code>ms-inventory/auditlog.json</code></a> are contained in the <code>content</code> key below.
+The (altered) contents of <a href="https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-inventory/auditlog.json"><code>ms-inventory/auditlog.json</code></a> are contained in the <code>content</code> key below.
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:422]: file exists on main, link to main branch

File: docs_src/phases/phase1.md [hunk 18]
```diff
@@ -447,7 +447,7 @@ Note that this API response now includes a financial transaction indicating what
 </p>
 
 <p>
-The (altered) contents of <a href="https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-ledger/ledger.json"><code>ms-ledger/ledger.json</code></a> are contained in the <code>content</code> key below.
+The (altered) contents of <a href="https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-ledger/ledger.json"><code>ms-ledger/ledger.json</code></a> are contained in the <code>content</code> key below.
```
Comment (lenny-intel) [inline docs_src/phases/phase1.md:450]: file exists on main, link to main branch

File: docs_src/phases/phase2.md [hunk 19]
```diff
@@ -189,9 +189,9 @@ The `ms-authentication` microservice contains an index of all cards, accounts, a
 
 First, navigate to the `ms-authentication` directory in the root of the repository. These three `.json` files dictate the `ms-authentication` service's behavior:
 
-- [`people.json`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-authentication/people.json)
-- [`accounts.json`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-authentication/accounts.json)
-- [`cards.json`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-authentication/cards.json)
+- [`people.json`](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-authentication/people.json)
+- [`accounts.json`](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-authentication/accounts.json)
+- [`cards.json`](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-authentication/cards.json)
```
Comment (lenny-intel) [inline docs_src/phases/phase2.md:194]: files exists on main, link to main branch

File: docs_src/references.md [hunk 20]
```diff
@@ -4,24 +4,24 @@
...
-| Controller Board Status | `as-controller-board-status` | [https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-controller-board-status](https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-controller-board-status) |
-| Vending                 | `as-vending`                 | [https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-vending](https://github.com/intel-iot-devkit/automated-checkout/blob/master/as-vending)                                 |
+| Controller Board Status | `as-controller-board-status` | [https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/as-controller-board-status](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/as-controller-board-status) |
+| Vending                 | `as-vending`                 | [https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/as-vending](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/as-vending)                                 |
```
Comment (lenny-intel) [inline docs_src/references.md:8]: Folders exists on main, link to main branch

File: docs_src/references.md [hunk 21]
```diff
@@ -4,24 +4,24 @@
...
-| Controller Board | `ds-controller-board`    | [https://github.com/intel-iot-devkit/automated-checkout/blob/master/ds-controller-board](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ds-controller-board) |
-| CV inference   | `ds-cv-inference`      | [https://github.com/intel-iot-devkit/automated-checkout/blob/master/ds-cv-inference](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ds-cv-inference)     |
+| Card Reader      | `ds-card-reader`         | [https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ds-card-reader](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ds-card-reader)           |
+| Controller Board | `ds-controller-board`    | [https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ds-controller-board](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ds-controller-board) |
+| CV inference   | `ds-cv-inference`      | [https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ds-cv-inference](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ds-cv-inference)     |
```
Comment (lenny-intel) [inline docs_src/references.md:16]: Folders exists on main, link to main branch

File: docs_src/references.md [hunk 22]
```diff
@@ -4,24 +4,24 @@
...
-| Inventory      | `ms-inventory`           | [https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-inventory](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-inventory)           |
-| Ledger         | `ms-ledger`              | [https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-ledger](https://github.com/intel-iot-devkit/automated-checkout/blob/master/ms-ledger)                 |
+| Authentication | `ms-authentication`      | [https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-authentication](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-authentication) |
+| Inventory      | `ms-inventory`           | [https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-inventory](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-inventory)           |
+| Ledger         | `ms-ledger`              | [https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-ledger](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/ms-ledger)                 |
```
Comment (lenny-intel) [inline docs_src/references.md:24]: Folders exists on main, link to main branch

File: docs_src/index.md [hunk 23]
```diff
@@ -126,21 +126,21 @@ docker ps --format 'table{{.Image}}\t{{.Status}}'
...
     | eclipse-mosquitto:2.0.14                             | Up 52 seconds     |
-    | automated-checkout/as-vending:dev                    | Up 52 seconds     |
-    | automated-checkout/ms-inventory:dev                  | Up 52 seconds     |
-    | automated-checkout/ds-controller-board:dev           | Up 52 seconds     |
-    | automated-checkout/ms-authentication:dev             | Up 55 seconds     |
+    | automated-vending/as-vending:dev                    | Up 52 seconds     |
+    | automated-vending/ms-inventory:dev                  | Up 52 seconds     |
+    | automated-vending/ds-controller-board:dev           | Up 52 seconds     |
+    | automated-vending/ms-authentication:dev             | Up 55 seconds     |
     | edgexfoundry/device-mqtt:2.2.0                       | Up 53 seconds     |
```
Comment (lenny-intel) [inline docs_src/index.md:135]:
```suggestion
    | edgexfoundry/device-mqtt:3.0.0                      | Up 53 seconds     |
```

File: docs_src/index.md [hunk 24]
```diff
@@ -126,21 +126,21 @@ docker ps --format 'table{{.Image}}\t{{.Status}}'
...
-    | automated-checkout/ds-card-reader:dev                | Up 53 seconds     |
-    | automated-checkout/as-controller-board-status:dev    | Up 52 seconds     |
+    | automated-vending/ds-card-reader:dev                | Up 53 seconds     |
+    | automated-vending/as-controller-board-status:dev    | Up 52 seconds     |
     | edgexfoundry/core-command:2.2.0                      | Up About a minute |
     | edgexfoundry/core-data:2.2.0                         | Up About a minute |
     | edgexfoundry/core-metadata:2.2.0                     | Up About a minute |
     | edgexfoundry/support-notifications:2.2.0             | Up About a minute |
```
Comment (lenny-intel) [inline docs_src/index.md:141]:
```suggestion
    | edgexfoundry/core-command:3.0.0                      | Up About a minute |
    | edgexfoundry/core-data:3.0.0                           | Up About a minute |
    | edgexfoundry/core-metadata:3.0.0                        | Up About a minute |
    | edgexfoundry/support-notifications:3.0.0                | Up About a minute |
```

File: docs_src/index.md [hunk 25]
```diff
@@ -126,21 +126,21 @@ docker ps --format 'table{{.Image}}\t{{.Status}}'
 
     | IMAGE                                                | STATUS            |
     |------------------------------------------------------|-------------------|
-    | automated-checkout/ms-ledger:dev                     | Up 53 seconds     |
+    | automated-vending/ms-ledger:dev                     | Up 53 seconds     |
     | eclipse-mosquitto:2.0.14                             | Up 52 seconds     |
```
Comment (lenny-intel) [inline docs_src/index.md:130]:
```suggestion
    | eclipse-mosquitto:2.0.15                             | Up 52 seconds     |
```

File: docs_src/index.md [hunk 26]
```diff
@@ -126,21 +126,21 @@ docker ps --format 'table{{.Image}}\t{{.Status}}'
...
     | edgexfoundry/device-mqtt:2.2.0                       | Up 53 seconds     |
-    | automated-checkout/ds-card-reader:dev                | Up 53 seconds     |
-    | automated-checkout/as-controller-board-status:dev    | Up 52 seconds     |
+    | automated-vending/ds-card-reader:dev                | Up 53 seconds     |
+    | automated-vending/as-controller-board-status:dev    | Up 52 seconds     |
     | edgexfoundry/core-command:2.2.0                      | Up About a minute |
     | edgexfoundry/core-data:2.2.0                         | Up About a minute |
     | edgexfoundry/core-metadata:2.2.0                     | Up About a minute |
     | edgexfoundry/support-notifications:2.2.0             | Up About a minute |
     | edgexfoundry/consul:1.10.10                          | Up About a minute |
```
Comment (lenny-intel) [inline docs_src/index.md:142]:
```suggestion
    | edgexfoundry/consul:1.15.2                          | Up About a minute |
```

File: docs_src/index.md [hunk 27]
```diff
@@ -126,21 +126,21 @@ docker ps --format 'table{{.Image}}\t{{.Status}}'
...
-    | automated-checkout/ds-cv-inference:dev               | Up 51 seconds     |
+    | automated-vending/ds-cv-inference:dev               | Up 51 seconds     |
     | redis:6.2-alpine                                     | Up About a minute |
```
Comment (lenny-intel) [inline docs_src/index.md:144]:
```suggestion
    | redis:7.0.11-alpine                                     | Up About a minute |
```

File: docs_src/index.md [hunk 28]
```diff
@@ -155,42 +155,42 @@ Then, navigate to the following Portainer URL and create an admin account:
...
 
 ## General Guidance
 
-After completing the steps in the Getting Started section, it may be helpful to read through the remainder of this document for some further guidance on the Automated Checkout reference implementation.
+After completing the steps in the Getting Started section, it may be helpful to read through the remainder of this document for some further guidance on the Automated Vending reference implementation.
 
 ### How to use the Compose Files
 
-The docker-compose files are segmented to allow for fine control of physical and simulated devices, as well as allowing you the choice of running Portainer. Use the [`makefile`](https://github.com/intel-iot-devkit/automated-checkout/blob/master/Makefile) to manage the various compose files:
+The docker-compose files are segmented to allow for fine control of physical and simulated devices, as well as allowing you the choice of running Portainer. Use the [`makefile`](https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/Makefile) to manage the various compose files:
```
Comment (lenny-intel) [inline docs_src/index.md:178]: file exists on main. Link to main

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: docs_src/phases/phase1.md [hunk 29]
```diff
@@ -2,9 +2,9 @@
 
 ## Overview
 
-This document aims to help provide a simple introduction to how we interact with the various microservices in the Automated Checkout reference implementation, as well as demonstrating how EdgeX command API's are leveraged.
+This document aims to help provide a simple introduction to how we interact with the various microservices in the Automated Vending reference implementation, as well as demonstrating how EdgeX command API's are leveraged.
 
-The steps in this guide show how to simulate the automated checkout workflow using [`curl`](https://github.com/curl/curl) REST API calls. It is a step-by-step walkthrough with specific commands to run on the command line.
+The steps in this guide show how to simulate the automated Vending workflow using [`curl`](https://github.com/curl/curl) REST API calls. It is a step-by-step walkthrough with specific commands to run on the command line.
```
Comment (ejlee3) [inline docs_src/phases/phase1.md:7]:
```suggestion
The steps in this guide show how to simulate the Automated Vending workflow using [`curl`](https://github.com/curl/curl) REST API calls. It is a step-by-step walkthrough with specific commands to run on the command line.
```

**Review by ejlee3 — CHANGES_REQUESTED** [review]

File: docs_src/automated-vending-services/device_services.md [hunk 30]
```diff
@@ -12,7 +12,7 @@ The **Automated Checkout** reference implementation utilizes three device servic
 
 ### Card reader description
 
-The `ds-card-reader` device service is an EdgeX device service that allows a USB-based RFID card reader to grant access to the Automated Checkout. At a high level, this device service is responsible for discovering a specific card reader device, watching for input from that device, parsing that input, and then forwarding the input into the EdgeX framework.
+The `ds-card-reader` device service is an EdgeX device service that allows a USB-based RFID card reader to grant access to the Automated Vending. At a high level, this device service is responsible for discovering a specific card reader device, watching for input from that device, parsing that input, and then forwarding the input into the EdgeX framework.
```
[the remark this answers is not in the document]
Comment (vli11) [reply docs_src/automated-vending-services/device_services.md:15]: we do need device to be authenticated to give information to automated vending. @brian-intel what do you think?

[the remark this answers is not in the document]
Comment (lenny-intel) [reply docs_src/automated-vending-services/device_services.md:15]: The only atuh is auth of the user. The way AV Auth works is the card is read by this service and then the Auth service validates the card number and grants/denies access to the cooler.

So maybe this is better?
```suggestion
The `ds-card-reader` device service is an EdgeX device service that allows a USB-based RFID card reader to read user's card for accessing to the Automated Vending. At a high level, this device service is responsible for discovering a specific card reader device, watching for input from that device, parsing that input, and then forwarding the input thru the EdgeX framework to the ms-authentication service which authorizes/denies access to the Automated Vending.
```

File: docs_src/configuration.md (hunk 6 above)
Comment (lenny-intel) [reply docs_src/configuration.md:10]: Right, They are not in the current as this is just an example of how to change configuration via env overrides.

File: docs_src/configuration.md [hunk 31]
```diff
@@ -58,19 +42,15 @@ The following items can be configured via the `DriverConfig` section of the serv
...
-`MQTTBrokerInfo`
+`device-mqtt.environment`
 
-- `Schema` - Data schema type, aka protocol
-- `Host` - Host name of the response MQTT Broker
-- `Port` - Port number of the response MQTT Broker
-- `Qos` - Quality of service agreement between sender and receiver
-- `KeepAlive` - Keep alive duration for the response MQTT Broker
-- `ClientId` - Client ID for the response MQTT Broker
-- `ResponseTopic` - Subscribe topic for the response MQTT Broker
```
Comment (lenny-intel) [inline docs_src/configuration.md:73]: The purpose of keeping these is an example if user wants to also change them. Please restore them in env override format.

File: docs_src/configuration.md [hunk 32]
```diff
@@ -58,19 +42,15 @@ The following items can be configured via the `DriverConfig` section of the serv
...
+This reference implementation uses the [MQTT Device Service](https://github.com/edgexfoundry/device-mqtt-go) from EdgeX with custom device profiles. These device profiles YAML files are located [here](https://github.com/intel-retail/automated-vending/tree/main/res/device-mqtt/profiles/inference-mqtt-device-profile.yml) and are volume mounted into the device service's running Docker container.
 
-The following items can be configured via `MQTTBrokerInfo` section of the service's [configuration.toml](https://github.com/intel-iot-devkit/automated-checkout/blob/master/res/device-mqtt/configuration.toml) file. All values are strings.
+The following items can be configured via `device-mqtt.environment` section of the service's [docker-compose.edgex.yml](https://github.com/intel-retail/automated-vending/tree/main/docker-compose.edgex.yml) file.
```
Comment (lenny-intel) [inline docs_src/configuration.md:47]:
```suggestion
The following `MQTTBrokerInfo` configuration items can be configured via `device-mqtt.environment` section of the service's [docker-compose.edgex.yml](https://github.com/intel-retail/automated-vending/tree/main/docker-compose.edgex.yml) file.
```

File: docs_src/configuration.md [hunk 33]
```diff
@@ -58,19 +42,15 @@ The following items can be configured via the `DriverConfig` section of the serv
...
 
-`MQTTBrokerInfo`
+`device-mqtt.environment`
```
Comment (lenny-intel) [inline docs_src/configuration.md:49]:
```suggestion
`MQTTBrokerInfo via device-mqtt.environment`
```

File: docs_src/automated-vending-services/device_services.md [hunk 34]
```diff
@@ -12,7 +12,7 @@ The **Automated Checkout** reference implementation utilizes three device servic
 
 ### Card reader description
 
-The `ds-card-reader` device service is an EdgeX device service that allows a USB-based RFID card reader to grant access to the Automated Checkout. At a high level, this device service is responsible for discovering a specific card reader device, watching for input from that device, parsing that input, and then forwarding the input into the EdgeX framework.
+The `ds-card-reader` device service is an EdgeX device service that allows a USB-based RFID card reader to read user's card for accessing to the Automated Vending. At a high level, this device service is responsible for discovering a specific card reader device, watching for input from that device, parsing that input, and then forwarding the input through the EdgeX framework to the ms-authentication service which authorizes/denies access to the Automated Vending.
```
Comment (brian-intel) [inline docs_src/automated-vending-services/device_services.md:15]:
```suggestion
The `ds-card-reader` device service is an EdgeX device service that allows a USB-based RFID card reader to read the user's card and grant access to the Automated Vending system. At a high level, this device service is responsible for discovering a specific card reader device, watching for input from that device, parsing that input, and then forwarding the input through the EdgeX framework to the ms-authentication service which authorizes/denies access to the Automated Vending.
```

File: docs_src/configuration.md [hunk 35]
```diff
@@ -32,14 +16,14 @@ These overrides are placed in the target service's environment section of the co
       - /dev/input:/dev/input
     environment:
       DRIVERCONFIG_SIMULATEDEVICE: "false"
-      DRIVERCONFIG_DEVICESEARCHPATH: "/dev/input/event*"
+      DRIVERCONFIG__DEVICESEARCHPATH: "/dev/input/event*"
```
Comment (lenny-intel) [inline docs_src/configuration.md:19]: Only one underscore

```suggestion
      DRIVERCONFIG_DEVICESEARCHPATH: "/dev/input/event*"
```

**Review by brian-intel — APPROVED** [review]

File: docs_src/configuration.md [hunk 36]
```diff
@@ -1,29 +1,13 @@
...
-```yaml
-Driver_VID: "256" ** Good **
-Driver_PID: "26"  ** Good **
-
-DRIVER_VID: "256" ** BAD **
-driver_pid: "26"  ** BAD **
-```
-
-These overrides are placed in the target service's environment section of the compose file. Here is an example:
+The simplest way to change one of the configuration values described below is via the use of environment variable overrides in the docker compose file. The value of each configuration item in a service's configuration can be overridden with an environment variable specific to that item. The name of the environment variable is the path to the item in the configuration tree with underscores separating the nodes. The character case of each node in the environment variable name must match that found in the service's configuration. These overrides are placed in the target service's environment section of the compose file. Here is an example:
```
Comment (lenny-intel) [inline docs_src/configuration.md:10]: Nothing actually described below...
```suggestion
The simplest way to change one of the service's configuration values is via the use of environment variable overrides in the docker compose file. The value of each configuration item in a service's configuration can be overridden with an environment variable specific to that item. The name of the environment variable is the path to the item in the configuration tree with underscores separating the nodes. The  environment variable name is upper case version of the setting that found in the service's configuration. These overrides are placed in the target service's environment section of the compose file. Here is an example:
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**